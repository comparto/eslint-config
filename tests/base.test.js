const base = require('../src/rules/base')

test('base.js', () => {
  expect(base).toMatchSnapshot()
})
