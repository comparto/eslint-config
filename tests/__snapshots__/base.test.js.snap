// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`base.js 1`] = `
Object {
  "env": Object {
    "browser": true,
    "es6": true,
    "node": true,
  },
  "extends": Array [
    "eslint:recommended",
  ],
  "globals": Object {
    "Atomics": "readonly",
    "SharedArrayBuffer": "readonly",
  },
  "parser": "babel-eslint",
  "parserOptions": Object {
    "ecmaFeatures": Object {
      "jsx": true,
    },
    "ecmaVersion": 2018,
    "sourceType": "module",
    "warnOnUnsupportedTypeScriptVersion": true,
  },
  "rules": Object {
    "no-console": "warn",
    "prefer-const": "error",
  },
}
`;
